openmdao_book/features/building_blocks/components/external_code_implicit_comp.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<p>Need to think about this block. In old docs, it is tested, but it is in an untested markup block here. How can we test this to make sure it doesn't include any errors?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>This is actually tested because the code in question exists in the file <code>extcode_mach.py</code> and it is called from the code cell two below this block. Here are the lines where it is called: </p><pre class="ql-syntax" spellcheck="false">self.options['command_apply'] = [
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; sys.executable, 'extcode_mach.py', self.input_file, self.output_file,
&nbsp; &nbsp; &nbsp; &nbsp; ]
&nbsp; &nbsp; &nbsp; &nbsp; self.options['command_solve'] = [
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; sys.executable, 'extcode_mach.py', self.input_file, self.output_file,
&nbsp; &nbsp; &nbsp; &nbsp; ] 
</pre>

Reply:
<p>The results are tested below this </p>